streamer/pipeline_configuration.py

```diff
@@ -62,6 +62,33 @@ class ProtectionScheme(enum.Enum):
   CBCS = 'cbcs'
   """AES-128-CBC mode with pattern encryption."""
 
+class ProtectionSystem(enum.Enum):
+  WIDEVINE = 'Widevine'
+  FAIRPLAY = 'FairPlay'
+  PLAYREADY = 'PlayReady'
+  MARLIN = 'Marlin'
+  COMMON = 'CommonSystem'
+
+class EncryptionMode(enum.Enum):
+  WIDEVINE = 'widevine'
+  """Widevine key server mode"""
+
+  RAW = 'raw'
+  """Raw key mode"""
+
+class RawKeyConfig(configuration.Base):
+  """An object representing a list of keys for Raw key encryption"""
+
+  label = configuration.Field(str).cast()
+  """An arbitary string or a predefined DRM label like AUDIO, SD, HD, etc.
+  If not specified, indicates the default key and key_id."""
```

Comment:
I don't understand this part.  What do you mean by default key and key id?  Are you saying the associated key and key id from this raw key entry is a default applied to all tracks without their own label?  (AUDIO, SD, HD, etc?)

Reply:
Yes that's correct, similar wording to the shaka-packager [documentation](https://google.github.io/shaka-packager/html/tutorials/raw_key.html#raw-key-encryption-options) but I omitted the last sentence which maybe clarifies it better?

Reply:
Okay, now I think I understand.  How about we review docs at some point on our side for clarity, but I don't put you through any more trouble on this PR.  This seems fine for now.

Thank you again for your contribution!